cumulus/polkadot-parachain/src/service.rs

```diff
@@ -281,12 +282,13 @@ pub(crate) trait NodeSpec {
 				let transaction_pool = transaction_pool.clone();
 				let backend_for_rpc = backend.clone();
 
-				Box::new(move |deny_unsafe, _| {
+				Box::new(move |deny_unsafe: sc_rpc::DenyUnsafe, _| {
```

Comment:
Why is the type needed now in contrast to before?

Reply:
I think I've hit the autocomplete by mistake :D No longer needed since I was able to obtain the same by moving the initialization to the substrate builder